Add interactive forms to the k8s section

anthonydillon opened the pull request:
## Done
Add interactive forms to the k8s section. Added a contact us button in the hero section.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /kubernetes
- Click the contact us button
- Check the form matches the [copy doc](https://docs.google.com/document/d/1F2DrXEU_XtnxsTkSbBIRf2l0vAsTHOY-HYu2Ry_693c/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4590

webteam-app commented:
Starting demo at: https://www-ubuntu-com-canonical-websites-pr-4646.run.demo.haus/

pmahnke commented:
I think it is odd that there are 2 other's on the first question and not just one in the last position in the third column like on page 2

![image](https://user-images.githubusercontent.com/441217/52281045-1f8ffd80-2955-11e9-89d4-f9627184a665.png)

anasereijo commented:
- [ ] add modal to the following 3 buttons
<img width="1106" alt="screenshot 2019-02-05 at 15 33 09" src="https://user-images.githubusercontent.com/36884067/52283976-726cb380-295b-11e9-8b45-6605b3b062d7.png">


- [ ] remove extra space between sections
<img width="1046" alt="screenshot 2019-02-05 at 15 32 57" src="https://user-images.githubusercontent.com/36884067/52283993-7b5d8500-295b-11e9-9410-8e48e82bccab.png">

anthonydillon commented:
@pmahnke I was told to leave them like that. Check comments in the document.

@anasereijo Addressed your comments. The space is due to needed to show an Other input when you select it.
